src/ConfigAggregator.php

```diff
@@ -274,12 +277,18 @@ private function cacheConfig(array $config, $cachedConfigFile)
             return;
         }
 
-        $contents = sprintf(
-            self::CACHE_TEMPLATE,
-            get_class($this),
-            date('c'),
-            var_export($config, true)
-        );
+        $hasVarExporter = is_callable("Brick\VarExporter\VarExporter::export");
```

Comment:
There might be room for improvement here by using class_exists, but this code is not performance relevant so anything that does work with PHP 5.6 and does not produce too many side effects should do the trick.